mismi-s3/src/Mismi/S3/Commands.hs

```diff
@@ -53,57 +54,89 @@ module Mismi.S3.Commands (
-import           Mismi.S3.Amazonka (send, paginate, Env )
+import           Mismi.S3.Amazonka (Env, send, paginate)
 import           Mismi.S3.Data
 import           Mismi.S3.Internal
 import qualified Mismi.S3.Patch.Network as N
 import qualified Mismi.S3.Patch.PutObjectACL as P
 
-import           Network.AWS.S3 hiding (headObject, Bucket, bucket, listObjects)
+import           Network.AWS.Data.Body (RqBody (..), RsBody (..), toBody)
+import           Network.AWS.Data.Body (ChunkedBody (..), ChunkSize (..))
+import           Network.AWS.Data.Text (toText)
+import           Network.AWS.S3 (BucketName (..))
+import           Network.AWS.S3 (ETag)
+import           Network.AWS.S3 (GetObjectResponse, gorsBody)
+import           Network.AWS.S3 (HeadObjectResponse, horsContentLength)
+import           Network.AWS.S3 (ListObjects, loPrefix, loDelimiter, loMarker)
+import           Network.AWS.S3 (ListObjectsResponse, lorsContents, lorsCommonPrefixes)
+import           Network.AWS.S3 (lorsIsTruncated, lorsNextMarker)
+import           Network.AWS.S3 (MetadataDirective (..))
+import           Network.AWS.S3 (MultipartUpload, muKey, muUploadId, muInitiated)
+import           Network.AWS.S3 (Object, oKey)
+import           Network.AWS.S3 (ObjectKey (..))
+import           Network.AWS.S3 (Part)
+import           Network.AWS.S3 (ServerSideEncryption (..))
+import           Network.AWS.S3 (cMultipartUpload)
+import           Network.AWS.S3 (cmuParts, cmuServerSideEncryption)
+import           Network.AWS.S3 (cmursUploadId)
+import           Network.AWS.S3 (coMetadataDirective, coServerSideEncryption)
+import           Network.AWS.S3 (cpPrefix)
+import           Network.AWS.S3 (goRange)
+import           Network.AWS.S3 (lmursUploads)
+import           Network.AWS.S3 (lprsParts)
+import           Network.AWS.S3 (poServerSideEncryption)
+import           Network.AWS.S3 (uprsETag)
+import           Network.AWS.S3 (createMultipartUpload, abortMultipartUpload, listMultipartUploads)
+import           Network.AWS.S3 (completeMultipartUpload, completedMultipartUpload, completedPart)
+import           Network.AWS.S3 (uploadPart)
+import           Network.AWS.S3 (getObject, putObject, copyObject, deleteObject)
 import qualified Network.AWS.S3 as AWS
-import           Network.AWS.Data.Body
-import           Network.AWS.Data.Text
 
 import           P
 import           Prelude (($!))
 
-import           System.IO
-import           System.Directory
-import           System.FilePath hiding ((</>))
-import           System.Posix.IO
+import           System.IO (IO, IOMode (..), SeekMode (..))
+import           System.IO (hFileSize, hSetFileSize, withFile)
+import           System.Directory (createDirectoryIfMissing, doesFileExist)
+import           System.FilePath (FilePath, dropFileName)
+import           System.Posix.IO (OpenMode(..), openFd, closeFd, fdSeek, defaultFileFlags)
 import qualified "unix-bytestring" System.Posix.IO.ByteString as UBS
```

Comment:
Is this package import really really required? The current version of `unix` has the function [fdWrite](http://hackage.haskell.org/package/unix-2.7.1.0/docs/System-Posix-IO-ByteString.html#v:fdWrite) which is the only thing it is used for.